custom/conf/app.example.ini

```diff
@@ -1156,15 +1156,9 @@ LEVEL = Info
 ;; enable cors headers (disabled by default)
 ;ENABLED = false
 ;;
-;; scheme of allowed requests
-;SCHEME = http
-;;
-;; list of requesting domains that are allowed
+;; list of requesting origins that are allowed, eg: "https://*.example.com"
 ;ALLOW_DOMAIN = *
 ;;
-;; allow subdomains of headers listed above to request
-;ALLOW_SUBDOMAIN = false
```

Comment:
I'm not really a fan of backporting config "changes" as that's unexpected in a minor version.

Reply:
Nevertheless, I think we should also add one of these `config <x> is no longer used` warnings for these two (at best in main, and then backport to 1.21 as well).

Reply:
> I'm not really a fan of backporting config "changes" as that's unexpected in a minor version.

It's not "changes", but it's "bug fix", because these options have been no-op for long time, keeping them would only mislead users but don't bring any real benefit.


> Nevertheless, I think we should also add one of these `config <x> is no longer used` warnings for these two (at best in main, and then backport to 1.21 as well).

I think it should be handled by the config framework automatically in the future, to help users to find their low-level mistakes. At this moment, I don't like adding them one by one manually.